lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
Where should I implement this method `Token`? I implemented it as instance and class method of `FormulaWrapper` and it gave me an error.